db/db_impl.h

```diff
@@ -234,8 +234,10 @@ class DBImpl : public DB {
       const FlushOptions& options,
       const std::vector<ColumnFamilyHandle*>& column_families) override;
   virtual Status FlushWAL(bool sync) override;
-  bool TEST_WALBufferIsEmpty();
+  bool TEST_WALBufferIsEmpty(bool lock = false);
```

Comment:
The default should be true, so that it would not affect the code that is using it with no default parameter.

Reply:
It meant to be true by default. I even passed "false" in my test case. Thanks for pointing it out. It didn't break any test cases it seems... which means it isn't really used in concurrent scenarios.